geomstats/geometry/poincare_ball.py

```diff
@@ -203,7 +193,7 @@ def mobius_add(self, point_a, point_b, project_first=True):
         mobius_add : array-like, shape=[..., dim]
             Result of the Mobius addition.
         """
-        ball_manifold = PoincareBall(self.dim, scale=self.scale)
+        ball_manifold = PoincareBall(self.dim)
```

Comment:
unrelated, but maybe we can move this to init and call it `self._space`, so later it will simplify the coming up refactoring of `Connection`.

Reply:
done

Reply:
Oops - not done! It's not so straightforward as there is a bit of circularity since `PoincareBall.__init__` calls `PoincareBallMetric.__init__`. Shall I just leave it alone?

Reply:
ah! yes, for now we keep. This kind of circularity will finally disappear with the design changes discussed in the last hackathon.